### ISSUE TITLE
Difficulty reproducing ResNet20/CIFAR-10 Rel. BOPs from Table 2

### ISSUE THREAD

**Issue by dadyownes15** [microsoft/geta#16]
Hi GETA team,

I am trying to reproduce the GETA ResNet20/CIFAR-10 result from Table 2 of the paper. The paper reports 91.42% accuracy and 4.5 relative BOPs.

Using the closest released script I found, I can reproduce the accuracy quite closely, but I cannot reproduce the reported relative BOPs. I used:

```bash
python test_scripts/Qtest_resnet56_ablation.py \
  --model_name resnet20 \
  --dataset cifar10 \
  --batch_size 64 \
  --epochs 350 \
  --lr 0.1 \
  --lr_quant 0.0001 \
  --weight_decay 0.0001 \
  --sparsity 0.35 \
  --projection_start_step 10 \
  --projection_periods 7 \
  --projection_steps 35 \
  --pruning_start_step 45 \
  --pruning_periods 5 \
  --pruning_steps 30 \
  --lr_step 100 \
  --lr_gamma 0.1 \
  --variant sgd \
  --bit_reduction 2 \
  --min_bit_wt 4 \
  --max_bit_wt 16 \
  --mix_up 0 \
  --label_smooth 0 \
  --ablation qhesso \
  --seed {0,1,2}
```

Across three seeds, I get:

| Seed | Best acc | Compressed acc | Rel. BOPs |
|---:|---:|---:|---:|
| 0 | 91.70 | 91.58 | 8.44 |
| 1 | 91.53 | 91.30 | 8.46 |
| 2 | 91.51 | 91.44 | 8.39 |
| Mean | 91.58 | 91.44 | 8.43 |

## Calculating the BOPs

I used OTO to calculate the BOPs. More specifically:

```python
# This is the compressed model saved by the training script. No modifications.
compressed_model = torch.load(
    "cache/ResNet20_compressed.pt",
    weights_only=False,
)

oto_compressed = OTO(
    compressed_model,
    dummy_input=torch.rand(1, 3, 32, 32),
)

compressed_bops = oto_compressed.compute_bops(in_million=True)["total"]

full_model = model_to_quantize_model(resnet20_cifar10())

oto_full = OTO(
    full_model,
    dummy_input=torch.rand(1, 3, 32, 32),
)

full_macs = oto_full.compute_macs(in_million=True)["total"]
full_bops = full_macs * 32 * 32

relative_bops = 100 * compressed_bops / full_bops
```

Could you clarify the exact setup used for the Table 2 ResNet20/CIFAR-10 result? Are there additional defaults or hyperparameters needed beyond Table 7 and the released script defaults?

I appreciate the work and hope you can help me resolve this discrepancy.
